src/unix/cups/dests.rs

```diff
@@ -59,8 +58,8 @@ impl CupsDestT {
         return value;
     }
 
-    pub fn is_shared_duplex(&self) -> bool {
-        return self.num_options == 5;
+    pub fn is_valid(&self) -> bool {
+        return self.num_options != 5 || self.get_state() != 0;
```

Comment:
see https://github.com/talesluna/rust-printers/pull/42

why is it valid when there are exactly five options?! I don't get it...